ios-drag-drop.js

```diff
@@ -345,6 +337,35 @@ function _exposeIosHtml5DragDropShim(config) {
     } while((el = el.parentNode) && el !== doc.body);
   }
 
+  function elementIsDraggable(el){
+    // if an element is not draggable either explicitly or implicitly we can exit immediately
+    if(!el.draggable === true) return false;
```

Comment:
Well `el.draggable` isn't guaranteed to be a boolean. However [JavaScript operator precedence](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence) dictates that **!** has higher priority than **===**, so what we're doing is `if ((!el.draggable) === true)` and `(!el.draggable)` is guaranteed to be a boolean.